Update Reaque description (#4)
mozdevz/opensource: README.md

## <a name="R"> </a>R

> * [reaque](https://github.com/horaciocome1/reaque) - Repository for the Reaque android app. Reaque is an mobile tool where we can share and find things related to Mozambique - the people and things there. From reflections and culture, throught history and gastronomy, to tourim and even fashion, if it is something about there, you should share and find it in Reaque.

Reviewer: ```suggestion
* [reaque](https://github.com/horaciocome1/reaque) - Repository for the Reaque android app. Reaque is a mobile tool where you can share and find stories from Mozambique. From thoughts and culture, through history and gastronomy, to tourism and even fashion, if it is something about Mozambique, you should share or find it in Reaque.
```

Author: It looks fancier. Lets keep it like this
